dispatchers_test.go

```diff
@@ -790,11 +916,110 @@ func TestDispatchFromFlagsAndUseBuiltInArgs(t *testing.T) {
 	args := []string{""}
 	original := "FROM --platform=$BUILDPLATFORM busybox"
 
-	if err := from(&mybuilder, args, nil, flags, original); err != nil {
+	if err := from(&mybuilder, args, nil, flags, original, nil); err != nil {
 		t.Errorf("dispatchAdd error: %v", err)
 	}
 
 	if mybuilder.Platform != expectedPlatform {
 		t.Errorf("Expected %v, to match %v\n", expectedPlatform, mybuilder.Platform)
 	}
 }
+
+func TestParseCommandForHeredoc(t *testing.T) {
+	command, pipe, err := parseCommandForHeredoc("RUN python3 <<EOF", "EOF", 0)
+	require.NoError(t, err)
+	assert.Equal(t, command, "python3")
```

Comment:
`assert.Equal()` expects to be passed the expected value before the actual value, and if it isn't, the error message gets very confusing.  This has tripped me up more than a few times.

Reply:
This code is removed and not needed.